probs/probs.go

```diff
@@ -92,19 +92,19 @@ func AccountDoesNotExist(detail string) *ProblemDetails {
 
 // AlreadyRevoked returns a ProblemDetails with a AlreadyRevokedProblem and a 400 Bad
 // Request status code.
-func AlreadyRevoked(detail string, a ...any) *ProblemDetails {
+func AlreadyRevoked(detail string) *ProblemDetails {
 	return &ProblemDetails{
 		Type:       AlreadyRevokedProblem,
-		Detail:     fmt.Sprintf(detail, a...),
+		Detail:     detail,
```

Comment:
Okay, last question; we're losing the support for formatting in `probs` and we're keeping that support for `berrors`? Is the rationale here that we hardly ever make use of it with `probs` and we're almost always making use of it with `berrors`?